src/containerapp/azext_containerapp/_clients.py

```diff
@@ -47,6 +47,38 @@ class ContainerAppPreviewClient(ContainerAppClient):
 
 class ContainerAppsJobPreviewClient(ContainerAppsJobClient):
     api_version = PREVIEW_API_VERSION
+    LOG_STREAM_API_VERSION = "2023-11-02-preview"
```

Comment:
Why use api-version "2023-11-02-preview"?

Reply:
Sync offline, as these REST api are not ready in `2024-02-02-preview`, it is fine you use  `LOG_STREAM_API_VERSION = "2023-11-02-preview"` before they are available in `2024-02-02-preview`.

Reply:
Will revert this api version once the new CP is deployed.